production/apache/fedoracommunity.conf

```diff
@@ -30,7 +30,7 @@ WSGIRestrictStdout Off
 WSGIRestrictSignal Off
 WSGIPythonOptimize 2
 
-WSGIScriptAlias /packages /usr/share/fedoracommunity/production/apache/fedoracommunity.wsgi
```

Comment:
This file is actually not used in production. We are maintaining the production file in ansible so we could delete this one.

Reply:
I moved it to the top to replace it with `devel/docker/fedora-packages.conf` version, but.. it broke `docker-compose` build with
```
Step 9/10 : COPY ../../fedora-packages.conf /etc/httpd/conf.d/fedora-packages.conf
ERROR: Service 'web' failed to build: Forbidden path outside the build context: ../../fedora-packages.conf ()
```
So I just removed it. I added a diff commit to see that there are no missing parts in devel config.